src/SelectPicker/index.tsx

```diff
@@ -79,37 +81,39 @@ const SelectPicker = ({
   const getRadioButtons = (options: any) => {
     if (options) {
       return options.map((option: Option) => (
-        <SingleCard
-          multiSelect={multiSelect}
-          key={option.value}
-          checked={isSelected(option)}
-          onClick={() => onPickerClick(option)}
-          withImage={option.image}
-          borderColor={borderColor}
-        >
-          <input
-            type="radio"
-            style={{ display: 'none' }}
-            value={option.value}
-            name={name}
+        <ThemeProvider theme={theme}>
```

Comment:
You’re wrapping single cards instead of whole components. This does not have correct font in Storybook. Probably because of this.

Reply:
I see that the font doesn't work for this component, but can you explain what you mean?